sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -253,9 +253,8 @@ object SharedState extends Logging {
     val warehousePath = if (hiveWarehouseDir != null && sparkWarehouseOption.isEmpty) {
       // If hive.metastore.warehouse.dir is set and spark.sql.warehouse.dir is not set,
       // we will respect the value of hive.metastore.warehouse.dir.
-      sparkConf.set(WAREHOUSE_PATH.key, hiveWarehouseDir)
       logInfo(s"${WAREHOUSE_PATH.key} is not set, but $hiveWarehouseKey is set. Setting" +
-        s" ${WAREHOUSE_PATH.key} to the value of $hiveWarehouseKey ('$hiveWarehouseDir').")
```

Comment:
why do we remove ` ('$hiveWarehouseDir')`?

Reply:
it's not final yet and the final one will be print right after this at https://github.com/apache/spark/pull/31671/files#diff-45744d86661942d414a6e12f72e1d24f62b1db1ce1a5a21a60dfb52abf7611b0R274


```
Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir ('datalake').
```

log like the above looks  confusing